packages/design-system/src/components/OcTable/OcTable.vue

```diff
@@ -220,11 +220,11 @@ export default defineComponent({
       default: null
     },
     /**
-     * The ids of disabled data items. Null or an empty string/array for no disabled items.
+     * The array of disabled data items. Empty array for no disabled items.
      */
     disabled: {
-      type: [String, Array],
-      default: null
+      type: Array as PropType<any[]>,
+      default: () => []
```

Comment:
Since we can't type it properly, could you add a comment explaining that an item needs to have one property that matches `idKey` (default = `id`).